FirstProject/txt.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FirstProject
+{
+    internal class txt
```

Comment:
Znowu słaba nazwa, txt co to jest? Wygląda jakbyś walną w klawiaturę. Do tego nazwy klas piszemy z wielkiej litery